lib/cast.js

```diff
@@ -63,9 +63,7 @@ exports.schema = function (config) {
         return internals.any.valid(config);
     }
 
-    Hoek.assert(config === null, 'Invalid schema content:', config);
-
-    return internals.any.valid(null);
```

Comment:
Why was this line changed to pass the config instead of `null` like it was previously?